browser/ui/side_panel/BUILD.gn

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+source_set("public") {
+  sources = [ "brave_side_panel_utils.h" ]
+}
```

Comment:
I was unsure what to do here @bridiver, as I didn't want to make `tabs:impl` depend on `//brave/browser/ui` - is there something better I can do here?

Reply:
This seems like a reasonable workaround for the issues with side_panel for this PR, but I would change the target name to `side_panel`

Reply:
Sweet, renamed